src/app/blocks/context-save-block/context-save-block.component.ts

```diff
@@ -11,28 +12,52 @@ import {mappingUtility} from '../mapping-block/mapping-util';
 })
 export class ContextSaveBlockComponent extends BaseBlockComponent {
 
-  contextKey = 'saved';
+  key = 'saved';
```

Comment:
I've changed the key from "contextKey" to "key" because we are no longer  just dealing with context. 
The old contextKey will still work for existing blocks.